README.md

```diff
@@ -1,6 +1,8 @@
+[![View on Maven Central](https://img.shields.io/maven-central/v/co.touchlab/kotlinnativecocoapods)](https://search.maven.org/artifact/co.touchlab/kotlinnativecocoapods)
+
 # Kotlin Cocoapods Extended
 
-This plugin is a fork of the Kotlin Multiplatform plugin's cocoapods support. Cocoapods support bundled into the KMP 
+This plugin is a fork of the `rg.jetbrains.kotlin.native.cocopods` plugin. Cocoapods support bundled into the KMP 
```

Comment:
`rg.jetbrains.kotlin.native.cocopods` instead of `org.jetbrains.kotlin.native.cocopods`

Reply:
oops